base/applications/regedit/edit.c

```diff
@@ -1670,8 +1670,6 @@ BOOL ModifyValue(HWND hwnd, HKEY hKey, LPCWSTR valueName, BOOL EditBin)
         lRet = ERROR_SUCCESS; /* Allow editing of (Default) values which don't exist */
         type = REG_SZ;
         valueDataLen = 0;
-        stringValueData = NULL;
-        binValueData = NULL;
```

Comment:
Few questions:
1. Is `binValueData` a global variable (to this edit.c file at least)?
2. What happens to the (pointer) value stored in `binValueData` at exit of the `ModifyValue` function when `EditBin == FALSE`?